docs/static/ea-integration-tutorial.asciidoc

```diff
@@ -0,0 +1,185 @@
+* Configure Fleet to send data from Elastic Agent to Logstash
+* Create an Elastic Agent policy with the necessary integrations
+* Configure Logstash to use the elastic_integration filter plugin
+
+
+Logstash elastic-integration Filter Plugin Guide
+
+Overview
+The purpose of this guide is to walk through the steps necessary to configure Logstash to transform events
+collected by the Elastic Agent using our pre-built ingest node pipelines that normalize data to the Elastic
+Common Schema. This is possible with a new beta feature in Logstash known as the elastic-integration
+filter plugin.
+Using this new plugin, Logstash reads certain field values generated by the Elastic Agent that tells Logstash to
+fetch pipeline definitions from an Elasticsearch cluster which Logstash can then use to process events before
+sending them to thier configured destinations.
+
+Prerequisites/Requirements
+
+There are a few requirements needed to make this possible:
+
+* A working Elasticsearch Cluster
+* Fleet server
+* An Elastic Agent configured to send its output to Logstash
+* An Enterprise License
+* A user configured with the minimum required privileges
+
+This feature can also be used with a self-managed agent, but the appropriate setup and configuration details
+of using a self-managed agent will not be provided in this guide.
+
+Configure Fleet to send data from Elastic Agent to Logstash
+
+. For Fleet Managed Agent, go to Kibana and navigate to Fleet → Settings.
+
+Figure 1: fleet-output
+
+. Create a new output and specify Logstash as the output type.
+
+Figure 2: logstash-output
+
+. Add the Logstash hosts (domain or IP address/s) that the Elastic Agent will send data to.
+. Add the client SSL certificate and the Client SSL certificate key to the configuration.
+You can specify at the bottom of the settings if you would like to make this out the default for agent
+integrations. By selecting this option, all Elastic Agent policies will default to using this Logstash output
+configuration.
+. Click “Save and apply settings” in the bottom right-hand corner of the page.
+
+Create an Elastic Agent policy with the necessary integrations
+
+. In Kibana navigate to Fleet → Agent policies and click on “Create agent policy”.
+
+
+
+Figure 3: create-agent-policy
+. Give this policy a name, and then click on “Advanced options”.
+. Change the “Output for integrations” setting to the Logstash output you created in the last step.
+
+
+
+Figure 4: policy-output
+
+
+. Click “Create agent policy” at the bottom of the flyout.
+. The new policy should be listed on the Agent policies page now.
+. Click on the policy name so that we can start configuring an integration.
+. On the policy page, click “Add integration”. This will take you to the integrations browser, where you
+can select an integration that will have data stream definitions (mappings, pipelines, etc.), dashboards,
+and data normalization pipelines that convert the source data into Elastic Common Schema.
```

Comment:
While an Elastic Integration _does_ (currently) contain these things, they are implementation details; when we elevate these details into our docs we encourage our users to depend on these details.

Instead I would recommend rephrasing this in general terms (the following needs workshopping for sure, but is a first-stab):

```suggestion
. On the policy page, click “Add integration”. This will take you to the integrations browser, where you
can select an integration that will have everything necessary to _integrate_ that data source with your other data in the Elastic stack.
```